### ISSUE TITLE
Problems with deployment

### ISSUE THREAD

**Issue by laughedelic** [probot/friction#25]
I've tried to deploy my bot to Glitch and to Now and I got the same problem: whenever it receives a webhook event, I see this in the logs:

```
04:40:02.772Z ERROR probot: fn.call is not a function (id=0f53b4ce-145d-11e8-98ef-a2af03c9091a)
  TypeError: fn.call is not a function
      at /home/nowuser/src/node_modules/express-async-errors/index.js:17:20
      at Layer.handle [as handle_request] (/home/nowuser/src/node_modules/express/lib/router/layer.js:95:5)
      at trim_prefix (/home/nowuser/src/node_modules/express/lib/router/index.js:317:13)
      at /home/nowuser/src/node_modules/express/lib/router/index.js:284:7
      at Function.process_params (/home/nowuser/src/node_modules/express/lib/router/index.js:335:12)
      at next (/home/nowuser/src/node_modules/express/lib/router/index.js:275:10)
      at /home/nowuser/src/node_modules/probot/lib/middleware/logging.js:33:5
      at /home/nowuser/src/node_modules/express-async-errors/index.js:17:20
      at Layer.handle [as handle_request] (/home/nowuser/src/node_modules/express/lib/router/layer.js:95:5)
      at trim_prefix (/home/nowuser/src/node_modules/express/lib/router/index.js:317:13)
      at /home/nowuser/src/node_modules/express/lib/router/index.js:284:7
      at Function.process_params (/home/nowuser/src/node_modules/express/lib/router/index.js:335:12)
      at next (/home/nowuser/src/node_modules/express/lib/router/index.js:275:10)
      at expressInit (/home/nowuser/src/node_modules/express/lib/middleware/init.js:40:5)
      at /home/nowuser/src/node_modules/express-async-errors/index.js:17:20
      at Layer.handle [as handle_request] (/home/nowuser/src/node_modules/express/lib/router/layer.js:95:5)
      at trim_prefix (/home/nowuser/src/node_modules/express/lib/router/index.js:317:13)
      at /home/nowuser/src/node_modules/express/lib/router/index.js:284:7
      at Function.process_params (/home/nowuser/src/node_modules/express/lib/router/index.js:335:12)
      at next (/home/nowuser/src/node_modules/express/lib/router/index.js:275:10)
      at query (/home/nowuser/src/node_modules/express/lib/middleware/query.js:45:5)
      at /home/nowuser/src/node_modules/express-async-errors/index.js:17:20
      at Layer.handle [as handle_request] (/home/nowuser/src/node_modules/express/lib/router/layer.js:95:5)
      at trim_prefix (/home/nowuser/src/node_modules/express/lib/router/index.js:317:13)
      at /home/nowuser/src/node_modules/express/lib/router/index.js:284:7
      at Function.process_params (/home/nowuser/src/node_modules/express/lib/router/index.js:335:12)
      at next (/home/nowuser/src/node_modules/express/lib/router/index.js:275:10)
      at Function.handle (/home/nowuser/src/node_modules/express/lib/router/index.js:174:3)
      at Function.handle (/home/nowuser/src/node_modules/express/lib/application.js:174:10)
      at Server.app (/home/nowuser/src/node_modules/express/lib/express.js:39:9)
      at Server.emit (events.js:160:13)
      at parserOnIncoming (_http_server.js:628:12)
      at HTTPParser.parserOnHeadersComplete (_http_common.js:112:17)
04:40:02.773Z  INFO probot: POST / 404 - 1.18 ms (id=0f53b4ce-145d-11e8-98ef-a2af03c9091a)
```

Same when I open the deployment URL, it fails with `GET / 404`. 

Notice, that with Now, I used the hack with concatenating the key with `\n`s. 

Here's my Glitch app: https://glitch.com/edit/#!/scalafmt-probot

**Comment by tcbyrd** [comment]
@laughedelic Thanks for reporting this. We've been seeing this on a few recent apps deployed with 5.0.0. For now can you try with Probot 4.1.0? We're going to look into this today.

**Comment by laughedelic** [comment]
@tcbyrd Thanks! This helped. It works on Glitch, but on Now I get

```
17:42:24.722Z ERROR probot:
  Promise {
    <rejected> Error: error:0906D06C:PEM routines:PEM_read_bio:no start line
      at Sign.sign (internal/crypto/sig.js:85:26)
      at Object.sign (/home/nowuser/src/node_modules/jwa/index.js:55:45)
      at Object.jwsSign [as sign] (/home/nowuser/src/node_modules/jws/lib/sign-stream.js:23:24)
      at Object.module.exports [as sign] (/home/nowuser/src/node_modules/jsonwebtoken/sign.js:186:16)
      at Robot.app (/home/nowuser/src/node_modules/probot/lib/github-app.js:12:16)
      at Robot.auth (/home/nowuser/src/node_modules/probot/lib/robot.js:157:61)
      at getInstallations (/home/nowuser/src/node_modules/probot/lib/plugins/stats.js:29:32)
      at refresh (/home/nowuser/src/node_modules/probot/lib/plugins/stats.js:22:33)
      at module.exports (/home/nowuser/src/node_modules/probot/lib/plugins/stats.js:9:24)
      at load (/home/nowuser/src/node_modules/probot/lib/index.js:55:5)
      at apps.concat.forEach.app (/home/nowuser/src/node_modules/probot/lib/index.js:66:45)
      at Array.forEach (<anonymous>)
      at Object.setup (/home/nowuser/src/node_modules/probot/lib/index.js:66:30)
      at pkgConf.then.pkg (/home/nowuser/src/node_modules/probot/bin/probot-run.js:51:10)
      at <anonymous> }
```

which is probably related to #17, although I did follow the workaround with concatenating the lines of the key.

**Comment by bkeepers** [comment]
> `TypeError: fn.call is not a function`

5.0.1. was released this morning with a fix for this.

> `Error: error:0906D06C:PEM routines:PEM_read_bio:no start line`

This means something is wrong with your private key. If you're deploying on Glitch, make sure you've uploaded your key from GitHub into `.data/private-key.pem` and set `PRIVATE_KEY_PATH=.data/private-key.pem` in `.env`.

**Comment by laughedelic** [comment]
Thanks, 5.0.1 solved the issue.

The other error occurred only on Now, Glitch works fine (modulo node version #24). I'm going to close this issue as solved and track #17 for the other problem.

Glitch is nice, but I'd still like to deploy on Now, because it's very easy to do from the console or CI. glitch-deploy which requires you github login/password for deployment and the web app that needs access to all your repos (including orgs and private ones) just to import a single one — it's too much!
